frontend/src/pages/Login.tsx

```diff
@@ -1,7 +1,40 @@
+import React from 'react';
+import GoogleLogin from 'react-google-login';
 import './Login.css';
+import axios from 'axios';
 
 function Login() {
-    return <h1>Login</h1>;
+    const onFailure = () => {
+        console.log('Invalid account');
+    };
+
+    const onSuccess = (googleData) => {
+        axios
```

Comment:
I would include the type of `googleData` here as I don't believe it can be inferred

Reply:
this causes some issues seeing as googleData can technically be of either type GoogleLoginResponse or GoogleLoginResponseOffline, with our use of the login box we shouldnt ever be getting GoogleLoginResponseOffline, however if we identity that as a possible type for googleData the function does not work seeing as tokenId is only a property of GoogleLoginResponse